src/sentry/static/sentry/app/views/groupDetails/shared/groupDetails.jsx

```diff
@@ -21,7 +21,11 @@ const GroupDetails = createReactClass({
   displayName: 'GroupDetails',
 
   propTypes: {
+<<<<<<< HEAD
```

Comment:
Some merge markers made their way in here

Reply:
😱 thanks!